src/ErrorUtil/validateErrors.js

```diff
@@ -0,0 +1,36 @@
+const validateError = (
+  targetCase,
+  supportedCases,
+  erorrMessages,
+  originalText,
+) => {
+  const error = {
+    errors: [],
+  };
+
+  const isSupportedCase = supportedCases.includes(targetCase);
+
+  if (targetCase) {
+    if (!isSupportedCase) {
+      error.errors.push({
+        message: erorrMessages.invalidCase,
+      });
+    }
+  } else {
+    error.errors.push({
+      message: erorrMessages.noToCaseParam,
+    });
+  }
+
+  if (originalText === '') {
```

Comment:
```suggestion
  if (!originalText) {
```
You don't need to compare with an empty string